atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -169,11 +169,6 @@ public abstract class TransactionManagers {
 
     abstract Set<Schema> schemas();
 
-    @Value.Default
-    Consumer<Object> registrar() {
-        return resource -> {};
-    }
```

Comment:
This is an ABI break - this is probably going to be pretty disruptive. I think we should just keep this.